test/testHeadless.m

```diff
@@ -18,9 +18,12 @@
 % The width and height are specified to circumvent different DPIs in developer
 % machines. The float format reduces the probability that numerical differences
 % in the order of numerical precision disrupt the output.
-    extraOptions = {'width' ,'\figurewidth' ,...
-                    'height','\figureheight',...
-                    'floatFormat', '%8.6g'  ,...
+    extraOptions = {'width' ,'\figW', ...
```

Comment:
I will update in a moment. Strangely Travis failed on a single test. Will have a look...